reporter/src/main/kotlin/utils/FreemarkerTemplateProcessor.kt

```diff
@@ -120,6 +118,7 @@ class FreemarkerTemplateProcessor(
             "packages" to packages,
             "ortResult" to input.ortResult,
             "licenseTextProvider" to input.licenseTextProvider,
+            "LicenseView" to LicenseView,
```

Comment:
TBH I'm (positively) surprised that it works to not pass an instance here, but a class. And not even just the companion object of the class. But I assume you've verified that this indeed works as expected?

Reply:
It actually does *not* seem to work: https://dev.azure.com/oss-review-toolkit/ort/_build/results?buildId=4068&view=logs&j=0b98c9ad-b4dd-509f-97af-ff0d5e511683&t=d4a109d9-63b6-59b7-7349-067146a0a444&l=3600

Reply:
Maybe there's also a misunderstanding, when looking at the commit message:

> Use LicenseView enum directly

`LicenseView` is not an `enum`, and `CONCLUDED_OR_DECLARED_AND_DETECTED` etc. are not `enum` members, but those are constants in a companion object.

Reply:
> It actually does not seem to work

It does work. I just forgot change some test templates.